Siri/L4/Event.java

```diff
@@ -0,0 +1,15 @@
+public class Event extends Task {
+
+    String eventDate;
+
+    public Event(String description, String eventDate) {
+        super(description);
+        this.eventDate = eventDate;
+    }
+
+    @Override
+    public String toString() {
+        return "[E]" + "[" + this.getStatusIcon() + "]" + this.description + "(at:" + this.eventDate + ")";
```

Comment:
Since this line is quite long, any reason why you did not wrap it on a new line?